autoload/lightline/bufferline.vim

```diff
@@ -58,6 +73,8 @@ function! s:get_buffer_name(i, buffer, path)
   endif
   if s:enable_devicons == 1 && exists('*WebDevIconsGetFileTypeSymbol')
     let l:name = WebDevIconsGetFileTypeSymbol(fnamemodify(bufname(a:buffer), ':t')) . ' ' . l:name
+  elseif s:enable_devicons == 1 && exists('g:loaded_devicons')
```

Comment:
Let me know if this flag should be put separately, but to me I think it's a good idea to use the same flag and implement a fallback instead. So this should take the vim-devicons first then nvim-web-devicons